targettests/Remote-Sim/observe_async.cpp

```diff
@@ -22,7 +20,7 @@ struct ansatz {
     cudaq::qvector q(2);
     x(q[0]);
     ry(theta, q[1]);
-    x<cudaq::ctrl>(q[1], q[0]);
+    cx(q[1], q[0]);
```

Comment:
Same and with all the rest below.

It's a little more work to do it correctly, but we'll run the different syntax of the tests in all the proper modes.

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1435/commits/4ad7ecd84a4f0d7c8dac820095463a5234365331